src/radio-group/__tests__/radio-group.test.tsx

```diff
@@ -110,6 +110,31 @@ describe('items', () => {
     expect(onChange).not.toHaveBeenCalled();
   });
 
+  test('can be readOnly', () => {
```

Comment:
The test name does not reflect what the test is doing. Suggestion "Adds aria-readonly when readOnly is set to true".

Reply:
agree! updated